tests/TestCase/Validation/ValidationTest.php

```diff
@@ -3172,4 +3172,19 @@ public function testHexColor()
         $this->assertFalse(Validation::hexColor('#fff'));
         $this->assertFalse(Validation::hexColor('ffffff'));
     }
+
+    /**
+     * Test IBAN
+     */
+    public function testIban()
+    {
+        $this->assertTrue(Validation::iban('AD1200012030200359100100'));
```

Comment:
Could this include all of these:
https://www.iban.com/structure.html

listed as IBAN SEPA and IBAN countries?

Reply:
Done